src/features/redux/cart/cartSlice.ts

```diff
@@ -0,0 +1,109 @@
+import { createSlice, SliceCaseReducers } from "@reduxjs/toolkit";
+
+export interface IAddress {
+  setShowNewAddressForm(showNewAddressForm: boolean): void;
+  showNewAddressForm: boolean;
+  showOrder: boolean;
+  setShowOrder(showOrder: boolean): void;
+  allAddresses: Array<Address>;
+  setAllAddresses(allAddresses: Array<Address>): void;
+}
+
+export interface INum {
+  number: string;
+  numberTitle: string;
+}
+
+export interface INumber {
+  setShowNewNumberForm(showNewNumberForm: boolean): void;
+  showNewNumberForm: boolean;
+  showOrder: boolean;
+  setShowOrder(showOrder: boolean): void;
+  allNumbers: Array<PhoneNumber>;
+  setAllNumbers(allNumbers: Array<PhoneNumber>): void;
+}
+
+export interface AddressFormDto {
+  addressTitle: string;
+  addresses: string;
+};
+
+export type Item = {
```

Comment:
```suggestion
export type TItem = {
```

Add prefixes for all types and interfaces u r using (Address, Time etc)